modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/Evaluator.java

```diff
@@ -1,28 +0,0 @@
-/*
- * Licensed to Elasticsearch under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package org.elasticsearch.index.rankeval;
-
-import org.elasticsearch.common.io.stream.NamedWriteable;
-import org.elasticsearch.search.SearchHit;
-
-public interface Evaluator extends NamedWriteable {
```

Comment:
I removed this in favour of making RankedListQualityMetric an abstract base class, we can re-introduce this if we need a common interface.